passport.md

```diff
@@ -253,108 +245,75 @@ Route::group([
 });
 ```
 
-<a name="issuing-access-tokens"></a>
-## Issuing Access Tokens
+<a name="authorization-code-grant"></a>
+## Authorization Code Grant
 
 Using OAuth2 via authorization codes is how most developers are familiar with OAuth2. When using authorization codes, a client application will redirect a user to your server where they will either approve or deny the request to issue an access token to the client.
 
-<a name="managing-clients"></a>
-### Managing Clients
-
-First, developers building applications that need to interact with your application's API will need to register their application with yours by creating a "client". Typically, this consists of providing the name of their application and a URL that your application can redirect to after users approve their request for authorization.
-
-<a name="the-passportclient-command"></a>
-#### The `passport:client` Command
+To get started, we need to instruct Passport how to return our "authorization" view. Remember, Passport is a headless OAuth2 library. If you would like a frontend implementation of Laravel's OAuth features that are already completed for you, you should use an [application starter kit](/docs/{{version}}/starter-kits).
 
-The simplest way to create a client is using the `passport:client` Artisan command. This command may be used to create your own clients for testing your OAuth2 functionality. When you run the `client` command, Passport will prompt you for more information about your client and will provide you with a client ID and secret:
+All the authorization view's rendering logic may be customized using the appropriate methods available via the `Laravel\Passport\Passport` class. Typically, you should call this method from the `boot` method of your application's `App\Providers\AppServiceProvider` class. Passport will take care of defining the `/oauth/authorize` route that returns this view:
 
-```shell
-php artisan passport:client
-```
-
-**Redirect URLs**
+```php
+use Laravel\Passport\Passport;
 
-If you would like to allow multiple redirect URLs for your client, you may specify them using a comma-delimited list when prompted for the URL by the `passport:client` command. Any URLs which contain commas should be URL encoded:
+/**
+ * Bootstrap any application services.
+ */
+public function boot(): void
+{
+    Passport::authorizationView('auth.oauth.authorize');
```

Comment:
We probably need to instruct users what this view should actually contain and what it should do on approval / denial.

Reply:
Instruction has been added.